CentralComputing/Pod.cpp

```diff
@@ -29,35 +38,103 @@ void int_handler(int signum) {
 int pod(int argc, char** argv) {
 	
 	signal(SIGINT, int_handler);
-	vector<Sensor_Configuration> configs = vector<Sensor_Configuration>();
+	auto configs = parse_input(argc, argv);
+	state = new Pod_State();
+		
+	sensors = new Sensor_Package(std::get<1>(configs), std::get<0>(configs));
+	printf("Created sensor package\n");
+	thread sensor_thread(sensor_loop);
+	thread network_thread(network_loop);
+
+	usleep(50000);
+	raise(SIGINT);
+
+	sensor_thread.join();
+	network_thread.join();
+	delete sensors;	
+
+
+	return 0;	
+} 
+
+std::tuple<bool, vector<Sensor_Configuration>> parse_input(int argc, char** argv) {
```

Comment:
Want to change the name of this to `init_sensors` or something? because this does not only parse the input